src/data-interfaces/HoloFuelDnaInterface.js

```diff
@@ -232,10 +251,17 @@ const HoloFuelDnaInterface = {
 
     accept: async (transactionId) => {
       const transaction = await HoloFuelDnaInterface.transactions.getPending(transactionId)
-      await createZomeCall('transactions/receive_payments_pending')({ promises: transactionId })
+      const result = await createZomeCall('transactions/receive_payments_pending')({ promises: transactionId })
+
+      if (Object.entries(result).length > 1) throw new Error('Multiple accepted offers were returned, when only one was triggered. \n Here is the Tansaction Origin ID of requested accept payment, and the resulting payments accepted : ', transactionId, Object.entries(result))
```

Comment:
Hmm. This feels a little fishy to me. Is this something the dna does? Is it a bug? If so, can we create an issue for the dna code? If not a bug, we should handle multiple offers returned (maybe filter by transactionId)